### PR TITLE
Fix several bugs related to format specifiers

### PR DESCRIPTION
**Pull request by ian-howell**
I noticed a handful of errors related to format specifiers. This PR should fix those up and get unit test passing

### CODE REVIEW

File: dependencies/config/config_test.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func init() {
 	hostname, err = os.Hostname()
 
 	if err != nil {
-		fmt.Errorf("Could not get hostname", err)
+		_ = fmt.Errorf("Could not get hostname %v", err)
```
Comment (PiotrProkop) [inline dependencies/config/config_test.go]: Maybe better to remove the whole if statement and change it to:
```
Expect(err).NotTo(HaveOccurred())
```
?

Comment (ian-howell) [reply dependencies/config/config_test.go]: I agree. Fixed

**Comment by ian-howell** [comment]
@PiotrProkop Have you had a second to look at this?

**Comment by PiotrProkop** [comment]
LGTM

**Pull request merged**